java/jdbc/src/test/java/com/flipkart/vitess/jdbc/test/VitessJDBCUrlTest.java

```diff
@@ -103,5 +103,25 @@
         Assert.assertEquals("user", vitessJDBCUrl.getUsername());
     }
 
+    @Test public void testWithKeyspaceandCatalog() throws Exception {
+        Properties info = new Properties();
+        System.out.println("I am here");
+        VitessJDBCUrl vitessJDBCUrl =
+            new VitessJDBCUrl("jdbc:vitess://user:password@10.33.17.231:15991/vt_shipment/shipment",
+                info);
+        Assert.assertEquals(1, vitessJDBCUrl.getHostInfos().size());
+        Assert.assertEquals("vt_shipment",vitessJDBCUrl.getKeyspace());
+        Assert.assertEquals("shipment",vitessJDBCUrl.getCatalog());
+    }
+
+    @Test public void testWithKeyspace() throws Exception {
+        Properties info = new Properties();
+        VitessJDBCUrl vitessJDBCUrl =
+            new VitessJDBCUrl("jdbc:vitess://user:password@10.33.17.231:15991/vt_shipment",
+                info);
+        Assert.assertEquals(1, vitessJDBCUrl.getHostInfos().size());
+        Assert.assertEquals("vt_shipment",vitessJDBCUrl.getKeyspace());
+        Assert.assertEquals("vt_shipment",vitessJDBCUrl.getCatalog());
+    }
```

Comment:
Is the output @ line 124 and 114 correct, what is the difference? one compare the vitessJDBCUrl.getCatalog() to "shipment" and the other to "vt_shipment"